### DISCUSSION TITLE
Add option to open/attach calendar to body for prevent clipped overflow

### DISCUSSION THREAD

**Discussion by WebCimes** [easepick/easepick#189] [Ideas]
Hi,

In some cases, when the datepicker's parent container is set to `position:relative;` and its style has an `overflow:scroll` or `overflow:hidden`, the modal is clipped by the container when the modal is open.

Like this (the container is with the red border, and it set with `position:relative` and an `overflow`):
![image](https://github.com/easepick/easepick/assets/43851969/4086e4a8-9408-413f-8083-4cb80d11ffea)

Maybe it will be a nice feature if you can add an option to choose where to attach the modal (like actually / or / to the body with a "position:fixed") to avoid clipping the modal. (some picker like jquery datepicker are attach to the body)

And then the excepted behavior:
![image](https://github.com/easepick/easepick/assets/43851969/fb5d923e-38a6-4266-a69a-c280710c961c)

Thanks

**Comment by JoseAlmeidaCamenAi** [comment]
I ran into a similar issue, for now the only workaround I've found for my case is to add some CSS for easepick-wrapper, like:

.easepick-wrapper {
   position: fixed !important; 
}

Had to add !important to override the inline style on that wrapper. with that my modal now overflows horizontally within my scrollable sidebar / menu, and it does still open nicely lined up below the input field.

The inconvenience with this solution is that it doesn't scroll together with the input when you scroll the parent container, it just stays in place where it opened. 

In my case this was already the same behavior I had with another jquery-based daterangepicker that instead attaches to body like you mentioned, but we can probably work around that too by setting an event listener for 'scroll' on that parent container, and calling the .hide() method of easepick to close the calendar modal.
